### ISSUE TITLE
Unconventional Indentation Behavior

### ISSUE THREAD

**Issue by eugenkiss** [GitbookIO/slate-edit-list#32]
Please see the following video: http://recordit.co/3BQ4heTdK5

When indenting a list item all of its sub-items are indented as well. One could make the case that this is good behavior. However, it is at least unconventional. For example, in Google Docs only the current list item will be indented. If you want to increase/decrease indentation of several items, you would apply a text selection over the list items.

What do you think? Should slate-edit-list copy Google Docs' behavior?

**Comment by Soreine** [comment]
Indeed, and Facebook Notes also only indent one item. That's something we should do as well. It would get along with implementing operations on multi-items.

On a side note: This plugin is closer from Facebook Notes behavior, than Google Docs. In Google Docs, you can have one item with arbitrary depth.
